src/util/apiUtils.test.js

```diff
@@ -37,6 +37,8 @@ import {
 	groupDuotoneSetter,
 } from './apiUtils';
 
+const API_META_HEADER = 'X-Meta-Request-Headers';
```

Comment:
might be better to define this constant in `apiUtils.js` and then import it here